core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -694,6 +712,15 @@ class Main {
 
                 EMConfig.Algorithm.RW ->
                     Key.get(object : TypeLiteral<RandomWalkAlgorithm<WebIndividual>>() {})
+
+                EMConfig.Algorithm.BreederGA ->
+                    Key.get(object : TypeLiteral<BreederGeneticAlgorithm<WebIndividual>>() {})
+
+                EMConfig.Algorithm.CellularGA ->
+                    Key.get(object : TypeLiteral<CellularGeneticAlgorithm<WebIndividual>>() {})
+
+                EMConfig.Algorithm.OnePlusLambdaLambdaGA ->
+                    Key.get(object : TypeLiteral<OnePlusLambdaLambdaGeneticAlgorithm<WebIndividual>>() {})
```

Comment:
these key configurations for algorithms are getting long... not sure there is any easy way to simplify them for different problems. maybe could be refactoring in a new utility function outside EMConfig file (but done in a different PR)